lib/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/page_commands.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:seeds/v2/domain-shared/page_command.dart';
+
+class NavigateToReceiveDetails extends PageCommand {}
+
+class ShowTransactionFailSnackBar extends PageCommand {}
```

Comment:
dont tell the UI what to use to show a message 

`ShowTransactionFail` the the UI decide how it will show it. Its not up to the state to decide how the UI shows it. 

The state tells the UI what to show not how to show it. 
I believe i left this feedback somewhere else already.

Reply:
understood, fix